### DISCUSSION TITLE
Gameobject personal visibility handling

### DISCUSSION THREAD

**Discussion by ghost** [TrinityCore/TrinityCore#29701] [Q&A]
Anyone know how, or if it's possible, to handle gameobjects to be invisible after using for quest? Video of quest I am working on.

https://www.youtube.com/watch?v=D1n4VAfv4Fk

**Comment by CraftedRO** [comment]
maybe you have more luck on discord

**Reply by ghost** [reply]
Dealing with some personal issues right now and if I was on Discord I'd end up taking it out on people.

**Comment by mdX7** [comment]
its the "Spawn Tracking" tech blizzard is using for it. That allows per-player fields (its not researched much, so far only visibility + flags are known) per gameobject - it's nyi currently
